cluster-up/hack/common.sh

```diff
@@ -31,7 +31,7 @@ KUBEVIRT_DEPLOY_PROMETHEUS_ALERTMANAGER=${KUBEVIRT_DEPLOY_PROMETHEUS_ALERTMANAGE
 KUBEVIRT_DEPLOY_GRAFANA=${KUBEVIRT_DEPLOY_GRAFANA:-false}
 KUBEVIRT_CGROUPV2=${KUBEVIRT_CGROUPV2:-false}
 KUBEVIRT_DEPLOY_CDI=${KUBEVIRT_DEPLOY_CDI:-false}
-KUBEVIRT_DEPLOY_CDI_LATEST=${KUBEVIRT_DEPLOY_CDI_LATEST:-false}
+KUBEVIRT_CUSTOM_CDI_VERSION=${KUBEVIRT_CUSTOM_CDI_VERSION}
```

Comment:
Does this need a default or is it ok? What happens when not set?

Reply:
We'll just use whatever was in kubevirtci at that point
(which gets bumped periodically by these: https://github.com/kubevirt/kubevirtci/pull/911)

Reply:
I was going to question if we even need this but it does make the variable more discoverable IMHO so I'm cool with it.